Assist/Properties/Languages/Lang.vi-vn.resx

```diff
@@ -0,0 +1,199 @@
+  </data>
+  <data name="Navigation_ProgressionHover" xml:space="preserve">
+    <value>Tiến độ</value>
+  </data>
+  <data name="Navigation_SettingsHover" xml:space="preserve">
+    <value>Cài đặt</value>
+    <comment>Text on Settings Button Hover</comment>
+  </data>
+  <data name="Navigation_StoreHover" xml:space="preserve">
+    <value>Cửa hàng</value>
+    <comment>Text on Store Button Hover</comment>
+  </data>
+  <data name="NightMarket_EndsIn" xml:space="preserve">
+    <value>Kết thúc sau</value>
+  </data>
+  <data name="NightMarket_Name" xml:space="preserve">
+    <value>Chợ đêm</value>
+  </data>
+  <data name="ProfileLaunchControl_LaunchSettingsHover" xml:space="preserve">
+    <value>Cài đặt khởi động</value>
+    <comment>Text on Launch Settings Button Hover</comment>
+  </data>
+  <data name="ProfileLaunchControl_PlayLabel" xml:space="preserve">
+    <value>Chơi</value>
+    <comment>Text on Play Label</comment>
+  </data>
+  <data name="ProfileLaunchControl_SwitchProfileHover" xml:space="preserve">
+    <value>Đổi tài khoản</value>
+    <comment>Text on Switch Profile Button Hover</comment>
+  </data>
+  <data name="Progression_Battlepass" xml:space="preserve">
+    <value>BATTLEPASS</value>
+  </data>
+  <data name="Progression_Battlepass_Completed" xml:space="preserve">
+    <value>Hoàn thành!</value>
+  </data>
+  <data name="Progression_Battlepass_CurrTier" xml:space="preserve">
+    <value>Hiện tại: </value>
+  </data>
+  <data name="SettingsGeneral_LanguageChange" xml:space="preserve">
+    <value>Ngôn ngữ: </value>
+  </data>
+  <data name="SettingsGeneral_SelLang" xml:space="preserve">
+    <value>Chọn ngôn ngữ:</value>
+  </data>
+  <data name="SettingsGeneral_SoundSetting" xml:space="preserve">
+    <value>Âm lượng:</value>
+  </data>
+  <data name="SettingsGeneral_WindowSize" xml:space="preserve">
+    <value>Cỡ cửa sổ:</value>
+  </data>
+  <data name="SettingsTab_About" xml:space="preserve">
+    <value>Về ứng dụng</value>
+  </data>
+  <data name="SettingsTab_General" xml:space="preserve">
+    <value>Chung</value>
+  </data>
+  <data name="SettingsTab_VALSet" xml:space="preserve">
+    <value>các cài đặt cho VALORANT</value>
+  </data>
+  <data name="StorePopup_BackLabel" xml:space="preserve">
+    <value>Quay lại</value>
+  </data>
+  <data name="StorePopup_ChromaLabel" xml:space="preserve">
+    <value>Đa sắc</value>
+  </data>
+  <data name="StorePopup_LevelsLabel" xml:space="preserve">
+    <value>Cấp độ</value>
+  </data>
+  <data name="Store_FeaturedBundleLabel" xml:space="preserve">
+    <value>Bundle hiện tại</value>
+  </data>
+  <data name="Store_FeaturedDailyLabel" xml:space="preserve">
+    <value>Shop hàng ngày</value>
+  </data>
+  <data name="Support_ButtonText" xml:space="preserve">
+    <value>Ủng hộ tác giả!</value>
+  </data>
+  <data name="Support_Description" xml:space="preserve">
+    <value>Assits sẽ luôn miễn phí và mã nguồn mở. cảm ơn các bạn đã sử dụng &lt;3</value>
```

Comment:
I think full translation would be better. Check your spelling too, not `Assits` but rather `Assist`
Suggestion: ` Assist sẽ luôn miễn phí và mã nguồn mở. Với sự ủng hộ của các bạn, bọn mình sẽ có thể tiếp tục phát triển và thêm vào những chức năng độc đáo mới. Cảm ơn các bạn đã sử dụng &lt;3.
`

Reply:
Cảm ơn bạn.
Thanks